cli/azd/pkg/project/framework_service_docker.go

```diff
@@ -128,11 +129,12 @@ func (p *dockerProject) Build(
 			dockerOptions := getDockerOptionsWithDefaults(serviceConfig.Docker)
 
 			log.Printf(
-				"building image for service %s, cwd: %s, path: %s, context: %s)",
+				"building image for service %s, cwd: %s, path: %s, context: %s, buildArgs: %s)",
```

Comment:
@ellismg that's a really good point. A common use-case I see would be passing in an API key so that there can be some data fetched during the build process or accessing an authenticated package (npm, nuget, etc.) feed, so you don't want that in the logs.

I'll add it to this PR as I'd prefer we ship it right the first time.

Reply:
@ellismg I've implemented this, but I'm not sure it's the most efficient way to do it, since the existing redaction code was tied to how we execute commands, not how we write to `log`, so I had to adapt it.

Reply:
Thanks, @aaronpowell, this looks like a good approach given the setup we had!